src/main.rs

```diff
@@ -55,6 +59,21 @@ impl RsIdentify {
         self.dmi_values.get(field_name).unwrap()
     }
 
+    #[cfg(target_os = "freebsd")]
+    fn get_dmi_field(&mut self, field_name: &str) -> &Option<String> {
+        let field_name_cmd = field_name.replace('_', "-");
+        if !self.dmi_values.contains_key(field_name) {
+            let output = Command::new("dmidecode")
```

Comment:
handling a lack of `dmidecode` might be a good idea
unless we make it an explicit dependency on *BSD.

Reply:
Given that we can't really do much without it present, I think an explicit dependency is reasonable.